### PR TITLE
224 DX

### PR DESCRIPTION
**Pull request by yoanficadiere**
Adding some rules in eslint and use typescript path for import

### CODE REVIEW

**Review by XavierJp — COMMENTED** [review]
First look seems good to me but there is conflict with `main` + could not build in local & could not test. 

We can discuss it tmr

File: .eslintrc [hunk 1]
```diff
@@ -1,13 +1,61 @@
 {
-  "extends": "next",
   "rules": {
+    "no-console": [
+      "error",
+      {
+        "allow": ["warn", "error"]
+      }
+    ],
+    // no need for unique key in maps as we dont use react on client/front
```
Comment (XavierJp) [inline .eslintrc:9]: Either remove the comment or it is in the wrong place

File: cypress/e2e/dynamic-pages.cy.js [hunk 2]
```diff
@@ -15,13 +15,6 @@ siret.forEach((siret) => {
         expect(resp.status).to.eq(200);
       });
     });
-
```
Comment (XavierJp) [inline cypress/e2e/dynamic-pages.cy.js:18]: Why delete this test ?

File: jest.config.js [hunk 3]
```diff
@@ -1,5 +1,23 @@
-module.exports = {
+const { pathsToModuleNameMapper } = require('ts-jest');
```
Comment (XavierJp) [inline jest.config.js:1]: Are we force to use this ? Feels kinda hacky

Comment (XavierJp) [reply jest.config.js:1]: But again it is fine. it is only the tests.

**Review by XavierJp — APPROVED** [review]

File: jest.config.js (hunk 3 above)
Comment (yoanficadiere) [reply jest.config.js:1]: Yeah this dev-dependencie is very usefull to make jest able to supports all features of TypeScript

**Pull request merged**